### ISSUE TITLE
Bump Node.js from v12 to v14

### ISSUE THREAD

**Issue by msorens** [chef/automate#4698]
## Describe the issue
The version of Node.js in use is behind. It needs to be upgraded from 12 to 14 per the guidelines on their site (https://nodejs.org/en/about/releases/). Follow the methodology outlined in https://github.com/chef/automate/pull/2579.

## To Reproduce
NA

## Expected behavior
We are running a version of Node.js supported for production builds, as outlined at https://nodejs.org/en/about/releases/.

**Comment by SEAjamieD** [comment]
@msorens - I imagine that it is correct to assume core plans will need an update to 14 as well, similar to https://github.com/habitat-sh/core-plans/pull/3158 ?

**Comment by msorens** [comment]
Correct... take a look at the PR I mentioned above (https://github.com/chef/automate/pull/2579) which explains necessary work both _before_ and _after_ the PR here in the automate repo.

**Comment by SEAjamieD** [comment]
For posterity, I've opened PR's in the following repos to kick off the pre work required

chef core-plans repo: https://github.com/habitat-sh/core-plans/pull/3580 
chef-base-plans repo: https://github.com/chef-base-plans/node14/pull/2
